platforms/documentation/docs/src/docs/userguide/glossary.adoc

```diff
@@ -50,6 +50,65 @@ For more information, see the sections on <<dependency_configurations.adoc#sub:w
 +
 NOTE: The word "configuration" is an overloaded term that has a different meaning outside of dependency management.
 
+[[sub:cross_configuration]]
+Cross-Configuration ::
+See <<sub:cross_project_configuration,Cross-Project Configuration>>.
+
+[[sub:cross_project_configuration]]
+Cross-Project Configuration ::
```

Comment:
💭 In this change, this concept is referred to as "cross-configuration", "cross-project configuration", and "cross-configuration from the root project".  Should the docs standardize on a single way of referring to this?

Reply:
I updated it to "cross-project configuration" across the docs or a similar term.